### PR TITLE
chore: make codecov only care about newlines

### PR DESCRIPTION
**Pull request by moul**
CleanShot 2024-12-05 at 17 01 41

RFC

### CODE REVIEW

**Comment by Gno2D2** [comment]
I'm a bot that assists the Gno Core team in maintaining this repository. My role is to ensure that contributors understand and follow our guidelines, helping to streamline the development process.

The following requirements must be fulfilled before a pull request can be merged.
Some requirement checks are automated and can be verified by the CI, while others need manual verification by a staff member.

These requirements are defined in this [configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

## Automated Checks

 🔴 Maintainers must be able to edit this pull request
 🟢 The pull request head branch must be up-to-date with its base


## Manual Checks

*No manual checks match this pull request.*

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>Maintainers must be able to edit this pull request</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🔴 Requirement not satisfied
└── 🔴 Maintainer can modify this pull request

```
</blockquote></details>

<details><summary>The pull request head branch must be up-to-date with its base</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Head branch (moul-patch-6) is up to date with base (master): behind by 0 / ahead by 1

```
</blockquote></details>

</blockquote></details>



</blockquote></details>

**Comment by aeddi** [comment]
@moul did you see this comment? https://github.com/gnolang/gno/issues/3238#issuecomment-2516995329

TL;DR: Codecov prevents the stale bot from doing its job correctly because:
- the stale bot adds a `stale` label to an inactive PR
- Codecov is triggered by the label addition and updates its report comment even if there is no change
- this produces activity on the PR and makes the stale bot remove the `stale` label

**Comment by n2p5** [comment]
I believe this is no longer relevant. @moul should we close?

**Comment by moul** [comment]
@n2p5 I believe it is: https://github.com/gnolang/gno/pull/3578#issuecomment-2618877085

**Review by aeddi — APPROVED** [review]
LGTM 👍

**Review by thehowl — APPROVED** [review]
let's try it
